workflows/flows/assemble_study_tasks/assemble_samplesheets.py

```diff
@@ -136,19 +135,19 @@ def update_assembly_metadata(
 
 
 @task
-def update_assemblies_assemblers_from_samplesheet(
+def update_assemblies_and_contaminant_ref_assemblers_from_samplesheet(
```

Comment:
Kinda confusing name now, maybe:

```suggestion
def update_assemblers_and_contaminant_ref_of_assemblies_from_samplesheet(
```

Reply:
Java, is that you?